gamemodes/terrortown/gamemode/client/cl_vskin/default_skin.lua

```diff
@@ -1404,6 +1404,73 @@ function SKIN:PaintSearchbar(panel, w, h)
 	)
 end
 
+---
+-- @param Panel panel
+-- @param number w
+-- @param number h
+-- @realm client
+function SKIN:PaintTextAreaTTT3(panel, w, h)
```

Comment:
Why are both this and the function on line 1438 needed?
If these are both needed could you explain their purpose/differences in the function comments?